modules/client/core/app/config.ts

```diff
@@ -59,6 +58,8 @@ const startAngularApp = () => {
         };
       }
     });
+
+    window.state = $state
```

Comment:
make angular state service global so we can call it from react while we're mid migration